Source/NaturalLanguageClassifierV1/NaturalLanguageClassifier.swift

```diff
@@ -0,0 +1,239 @@
+            acceptType: "application/json"
+        )
+        
+        // execute REST request
+        Alamofire.request(request)
+            .authenticate(user: username, password: password)
+            .responseArray(dataToError: dataToError, path: ["classifiers"]) {
+                (response: Response<[ClassifierModel], NSError>) in
+                switch response.result {
+                case .Success(let classifiers): success(classifiers)
+                case .Failure(let error): failure?(error)
+                }
+        }
+    }
+    
+    /**
+     Sends data to create and train a classifier and returns information about the new
+     classifier. When the operation is successful, the status of the classifier is set
+     to "Training". The status must be "Available" before you can use the classifier.
+     
+     - parameter trainingMetadata:
+     - parameter trainingData:      The set of questions and their "keys" used to adapt a system to a domain (the ground truth).
+     - parameter failure: A function executed if an error occurs.
+     - parameter success: A function executed with the list of available standard and custom models.
+     */
+    public func createClassifier(
+        trainingMetadata: NSURL,
+        trainingData: NSURL,
+        failure: (NSError -> Void)? = nil,
+        success: ClassifierDetails -> Void) {
+        
+        // construct REST request
+        let request = RestRequest(
+            method: .POST,
+            url: serviceURL + "/v1/classifiers",
+            acceptType: "application/json"
+        )
+        
+        // execute REST request
+        Alamofire.upload(request,
+            multipartFormData: { multipartFormData in
+                multipartFormData.appendBodyPart(fileURL: trainingMetadata, name: "training_metadata")
+                multipartFormData.appendBodyPart(fileURL: trainingData, name: "training_data")
+            },
+            encodingCompletion: { encodingResult in
+                switch encodingResult {
+                case .Success(let upload, _, _):
+                    upload.authenticate(user: self.username, password: self.password)
+                    upload.responseObject(dataToError: self.dataToError) {
+                        (response: Response<ClassifierDetails, NSError>) in
+                        switch response.result {
+                        case .Success(let classifierDetails): success(classifierDetails)
+                        case .Failure(let error): failure?(error)
+                        }
+                    }
+                case .Failure:
+                    let failureReason = "Files could not be encoded as form data."
+                    let userInfo = [NSLocalizedFailureReasonErrorKey: failureReason]
+                    let error = NSError(domain: self.domain, code: 0, userInfo: userInfo)
+                    failure?(error)
+                    return
+                }
+            }
+        )
+    }
+    
+    /**
```

Comment:
This documentation can be updated as well. I think you get the point, so I'll stop mentioning it on other methods. Basically:
- Be careful about using "returns"
- Remove unnecessary whitespace
- Make sure the `success` parameter is correct for this particular method

On a side-note, I assume the whitespace came from VVDocumenter? I personally found that frustrating and dropped it awhile ago. Unfortunately, that extra whitespace makes it difficult to update the comments after making a change to the function signature. It also makes it a hassle if you use a long description and want to drop down to a new line.